.travis.yml

```diff
@@ -8,6 +8,8 @@ jobs:
   include:
     - python: 3.6
       env: TOX_ENV=format
+    - python: 3.6
```

Comment:
Can it also be done for 3.7 and 3.8?

Reply:
I don't think the runtime version matters. I believe running it in just 1 version is sufficient. Adding 2 more I think would just increase our travis runtime with no benefit.

Reply:
Make sense.